source/_integrations/cover.mqtt.markdown

```diff
@@ -421,7 +421,67 @@ cover:
 ```
 
 {% endraw %}
+### Full configuration using advanced templating
 
+The example below shows a full example how to setup a venetian blind, a blind with moveable slats, that tilts with a position change, that does not have a topic for tilt. In the example it takes the blind 6% of the movement for a full rotation of the slats.
```

Comment:
```suggestion
The example below shows a full example of how to setup a venetian blind which has a combined position and tilt topic. The blind in the example has moveable slats which tilt with a position change. In the example it takes the blind 6% of the movement for a full rotation of the slats.
```
Two questions:
- Can the slats really perform a full rotation, or is 6% the movement needed to fully open or close the slats?
- The slats will open or close during the initial 6% of position change, _only after the direction of movement is reversed_, is that right?